tests/post_training/pipelines/image_classification_timm.py

```diff
@@ -46,19 +46,30 @@ def prepare_model(self) -> None:
         timm_model = replace_timm_custom_modules_with_torch_native(timm_model)
         self.model_cfg = timm_model.default_cfg
         self.input_size = [1] + list(timm_model.default_cfg["input_size"])
+        self.dynamic_input_size = [-1] + list(timm_model.default_cfg["input_size"])
         self.dummy_tensor = torch.rand(self.input_size)
 
         if self.backend in PT_BACKENDS:
             self.model = timm_model
```

Comment:
What about the batch support for `PT_BACKENDS`?

Reply:
For Torch there is no need to do export, so this has no effect on Torch. 
However, for Torch quantization with batch_size is enabled as for a dataloader a new batch_size value is set.